dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Skills.Memory.VectorDB.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+    </PropertyGroup>
+    <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+
+    <PropertyGroup>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <AssemblyName>Microsoft.SemanticKernel.Skills.Memory.VectorDB</AssemblyName>
+        <RootNamespace>Microsoft.SemanticKernel.Skills.Memory.VectorDB</RootNamespace>
+        <TargetFramework>netstandard2.1</TargetFramework>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <!-- NuGet Package Settings -->
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <PackageId>Microsoft.SemanticKernel.Skills.Memory.VectorDB</PackageId>
+        <Title>Semantic Kernel - VectorDB Connector</Title>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.CSharp" VersionOverride="4.7.0" />
+        <PackageReference Include="Microsoft.Extensions.Logging.Console" />
+        <PackageReference Include="Newtonsoft.Json" VersionOverride="13.0.2" />
```

Comment:
Package versions must be specified in Directory.Packages.props file.

We'll want to replace Newtonsoft, but don't worry about it for this first version -- I know this sdk uses it for deserializing to `dynamic`, which System.Text.Json doesn't support.

Reply:
Okay will do, the new classes that were added I did use the System.Text.Json to limit Newtonsoft has much as I could.